lib/puppet/version.rb

```diff
@@ -72,6 +72,10 @@ def self.version=(version)
     @puppet_version = version
   end
 
+  def self.major_minor_version
+    self.version.sub(/\A(\d+\.\d+)\..*$/, '\1')
+  end
```

Comment:
There isn't much to consolidate - they are basically the same except that the name chosen for the method is different.
While we could use semantic puppet to parse, we also know that we are using a basic standard versioning scheme with three digits. If we change that then we need to change here as well.